tests/graphql_spec.rs

```diff
@@ -193,10 +232,14 @@ fn test_server_to_client_sdl() -> std::io::Result<()> {
   let specs = GraphQLSpec::cargo_read("tests/graphql");
 
   for spec in specs? {
-    let expected = spec.client_sdl;
-    let content = spec.server_sdl[0].as_str();
+    let expected = spec.find_source(SERVER_SDL);
+    let expected = expected.as_str();
+    let content = spec.find_source(CLIENT_SDL);
+    let content = content.as_str();
     let config = Config::from_sdl(content).to_result().unwrap();
+    // error is on the line below
     let actual = print_schema::print_schema((Blueprint::try_from(&config).unwrap()).to_schema());
```

Comment:
I'm trying to understand why this line is throwing an error; can I get some help here? :eyes:

Reply:
What's the error?

Reply:
```
thread 'test_server_to_client_sdl' panicked at 'called `Result::unwrap()` on an `Err` value: ValidationError([Cause { message: "No resolver has been found in the schema", description: None, trace: ["Query", "hello"] }])', tests/graphql_spec.rs:242:75
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
I've mentioned on [this comment](https://github.com/tailcallhq/tailcall/pull/602#discussion_r1386887309), but this error is appearing when executing `/tests/graphql/test-upstream.graphql`

Reply:
I've fixed it; thanks @tusharmath